packages/button/button.scss

```diff
@@ -130,21 +132,19 @@ $hover-elevation-distance: -0.25rem;
     }
 
     @include small-device {
-        @include jkl-text-style("compact/body");
-        font-weight: $bold-weight;
-        height: $button-height--compact;
+        font-size: $button-text-size--compact;
         line-height: $button-height--compact;
+        height: $button-height--compact;
```

Comment:
Er ikke 36px for lite med tanke på klikkflate for a11y?
Jeg trodde 48x48 var minium, men blir det aneledes når den blir 36x80? I mitt hode så er dette for lite for mobil.

Reply:
Tror det er greit så lenge én av dimensjonene er større enn 48px 🤔 Vi snakket litt frem og tilbake om dette for en stund siden husker jeg, men husker ikke alt som kom opp 😅

Ser at Material sine knapper er 36px høye, men er mulig de gjør noe magi med touch targets..